common/locales/en/pets.json

```diff
@@ -1,10 +1,14 @@
 {
     "pets": "Pets",
+    "activePet": "Active Pet",
+    "noActivePet": "No Active Pet",
```

Comment:
@crookedneighbor Would it be worthwhile to add a sanity check that all localized strings are used at least once?